### ISSUE TITLE
[FEATURE REQUEST] Indentation

### ISSUE THREAD

**Issue by flamendless** [coding-jackalope/Slab#17]
Sample use-case
```lua
Slab.Text("Position:")
Slab.Indent() --this is to indent succeeding 'text' (or event other element) calls
Slab.Text("x: " .. 0) 
Slab.Text("y: " .. 0)
Slab.Unindent() --the closing of Indent

Slab.Text("Position 2:")
Slab.Indent()
Slab.Text("x: " .. 1) 
Slab.Text("y: " .. 1)
```
to display:
```
Position:
    x: 0
    y: 0
Position 2:
    x: 1
    y: 1
```

**Comment by coding-jackalope** [comment]
Hi, thanks for the suggestion. I will have this implemented in the next update.

**Comment by coding-jackalope** [comment]
This feature has been implemented in v0.6.4. This commit can be found here a8b6b2a.
